app/assets/javascripts/admin/helpers/taxonomy_tree_helper.js

```diff
@@ -0,0 +1,46 @@
+(function () {
+    "use strict";
+    window.GOVUK = window.GOVUK || {};
+
+    window.GOVUK.taxonomyTreeHelper = {
+        ancestors: function ancestors(element) {
```

Comment:
This function, and the ones below it can probably be anonymous (e.g. `function(element) {`), as the name is given by the name of the value in the `taxonomyTreeHelper` object.

Reply:
I've fixed this now :)